docs/source/using/image-caching.adoc

```diff
@@ -4,39 +4,121 @@ include::variables.adoc[]
 :icons:
 :toc: macro
 :toc-title:
-:toclevels: 1
+:toclevels: 2
 
 toc::[]
 
 [[image-caching-overview]]
 == Overview
 
-To speed up provisioning of the OpenShift cluster and to minimize network traffic, the core OpenShift images can be cached on the host.
-This feature is considered experimental and needs to be explicitly enabled.
+To speed up the provisioning of the OpenShift cluster and to minimize network traffic, container images can be cached on the host.
+These images can then be imported into the running Docker daemon, either explicitly on request or implicitly during xref:../using/basic-usage.adoc#minishift-start-overview[`minishift start`].
+The following sections describe image caching and its configuration in more detail.
 
-[[image-chaching-configuration]]
-== Image Caching Configuration
+[[explicit-image-caching]]
+== Explicit Image Caching
 
-To enable image caching you use the xref:../command-ref/minishift_config_set#[`minishift config set`] command:
+{project} provides the `image` command together with its sub-commands to control the image caching behavior.
+To export and import images from the Docker daemon of the {project} VM, you need to run `minishift image export` respectively `minishift image import`.
+
+[[single-images]]
+=== Importing and Exporting Single Images
+
+Once the {project} VM is running, images can be explicitly exported from the Docker daemon by:
```

Comment:
I would replace "by" with "with the following" or remove it entirely.

Reply:
I removed it